CODEOWNERS

```diff
@@ -1,16 +1,29 @@
-/.github/ @neondatabase/developer-productivity
-/compute_tools/ @neondatabase/control-plane @neondatabase/compute
-/libs/pageserver_api/ @neondatabase/storage
-/libs/postgres_ffi/ @neondatabase/compute @neondatabase/storage
-/libs/proxy/ @neondatabase/proxy
-/libs/remote_storage/ @neondatabase/storage
-/libs/safekeeper_api/ @neondatabase/storage
+# Autoscaling
 /libs/vm_monitor/ @neondatabase/autoscaling
-/pageserver/ @neondatabase/storage
+
+# DevProd
+/.github/ @neondatabase/developer-productivity
+
+# Compute
 /pgxn/ @neondatabase/compute
-/pgxn/neon/ @neondatabase/compute @neondatabase/storage
+/vendor/ @neondatabase/compute
+/compute/ @neondatabase/compute
+/compute_tools/ @neondatabase/compute
```

Comment:
@clipperhouse and @neondatabase/control-plane let me know what do you think about removing control plane from here, but adding into `/libs/compute_api/`, because the latter is the shared public interface of the compute

Reply:
Yeah, that was my idea to add cplane into compute_api

I am also wondering about removing cplane from compute_tools. It's mostly internal compute code, but then it depends on how noisy it is. Both keeping and removing work for me. What do you think?

Reply:
It’s my sense that people from control plane haven’t had anything to add to these reviews? No problems are being prevented?

In such situations, I think it would be covered by e2e tests. Of course I don’t want to have an “we could have prevented this” situation but perhaps code review isn’t best way, esp if that code review is often uninteresting.

[Discussion](https://neondb.slack.com/archives/C03438W3FLZ/p1733335344711819).

Reply:
Based on the above discussion, I think switching to `/libs/compute_api/`is an appropriate choice. I imagine those changes will be rare, and should be given a look on our side.